sdk/src/types/block/context_input/block_issuance_credit.rs

```diff
@@ -53,4 +54,10 @@ pub(crate) mod dto {
             Self::new(value.account_id)
         }
     }
+
+    impl_serde_typed_dto!(
+        BlockIssuanceCreditContextInput,
+        BlockIssuanceCreditContextInputDto,
+        "block issuance credit input"
```

Comment:
```suggestion
        "block issuance credit context input"
```

Reply:
This is already done, isn't it? or GitHub is buggy again?

Reply:
Yeah now I also see it updated